config/prow/pj-on-kind.sh.from.oss.repo

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
```

Comment:
This an the original (?) https://github.com/knative/test-infra/blob/main/config/prow/pj-on-kind.sh should be merged.